components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -337,6 +365,13 @@ var NodeState = Mapping{
 				"refresh_interval": "5s"
 			},
 			 "analysis": {
+				"normalizer": {
+					"case_insensitive": {
+						"type": "custom",
+						"char_filter": [],
+						"filter": ["lowercase", "asciifolding"]
+					}
+				},
```

Comment:
Adding the `case_insensitive` normalizer. This change requires migration of the node-state index.